src/utils/RoleUtils.js

```diff
@@ -95,3 +95,7 @@ export function canSelectInstitution(currentUser, user) {
     canWriteUserInfo(currentUser, user)
   );
 }
```

Comment:
why is formatting canWriteInstitution and canReadInstitution different ... it seems to me that parenthessis is missing in 
```
hasRole(
    currentUser,
    ROLE.WRITE_ALL_ORGANIZATIONS ||
```

maybe ther should be `)` before `||` ?

Reply:
Shouldn't canWriteInstitution and canReadInstitution follow the same pattern just WRITE --> READ ?